ansible/vars/vars.yml

```diff
@@ -1,7 +1,7 @@
 ---
-siteuser: mano-aloe
+siteuser: valentine
 installpath: "/opt/{{ siteuser }}"
-gitrepo: "https://github.com/AreYouW/mano-aloe"
-backend_proto: "https"
-backend_ip: "manotomo.tk"
-backend_port: "443"
+gitrepo: "https://github.com/Manotomo-Alliance-Support-Squad/holo-valentine-2021"
+backend_proto: "http"
```

Comment:
Not https?

Reply:
It's because I don't want to issue a local cert for my testing vm. I can change it before I deploy to the actual site